switchboard/Switchboard.scala

```diff
@@ -0,0 +1,47 @@
+package switchboard
+
+import conf.Configuration
+import common._
+import switchServices.switch
+import switches.switchManager
+import play.api.GlobalSettings
+import play.api.{Logger, Application, GlobalSettings}
+import play.api.libs.json.Json
+
+trait NewSwitchboardLifecycle extends GlobalSettings with ExecutionContexts with Logging {
+
+  override def onStart(app: Application) {
+    super.onStart(app)
+    Jobs.deschedule("SwitchBoardRefreshJob")
```

Comment:
I just realized that `Jobs` comes from `common` and it's quite a complex class.

Can you try using `Akka.scheduler`

Should work like this
```
Akka.scheduler.schedule(0.seconds, 1.minute) { refresh() }
```